packages/tasks/src/snippets/python.ts

```diff
@@ -12,6 +12,22 @@ output = query({
     "parameters": {"candidate_labels": ["refund", "legal", "faq"]},
 })`;
 
+export const snippetZeroShotImageClassification = (model: ModelData): string =>
+	`def query(data):
+	with open(data["image_path"], "rb") as f:
+		img = f.read()
+	payload={
+		"parameters": data["parameters"],
+		"inputs": base64.b64encode(img).decode("utf-8")
+	}
+	response = requests.post(API_URL, headers=headers, json=payload)
+	return response.json()
+
+output = query({
+    "inputs": ${getModelInputSnippet(model)},
```

Comment:
I'm a little confused, aren't we passing the example input (which is the image path in the other file)?
if so, isn't there supposed to be an `image_path` key in the dict that is passed to the query function? 

if so it should be:
output = query({
    "image_path": ${getModelInputSnippet(model)},

Reply:
Good catch! Fixed it